_docs/features.md

```diff
@@ -0,0 +1,118 @@
+              Code hosted on GitHub
+            </p>
+          </div>
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Available under GPL License
+            </p>
+          </div>
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Growing community
+            </p>
+          </div>
+        </div>
+      <h5 class="u-title u-center u-spacerUp">Software Foundation</h5>
+        <div class="row">
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Made with popular Python and Django
+            </p>
+          </div>
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Default React and Redux frontend
+            </p>
+          </div>
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Django Rest Framework API
+            </p>
+          </div>
+        </div>
+      <h5 class="u-title u-center u-spacerUp">Easy Installation</h5>
+        <div class="row">
+          <div class="col-sm-3 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Option to install with Docker or pyenv
+            </p>
+          </div>
+          <div class="col-sm-3 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              A quickstart cookiecutter template for creating a game user interface
+            </p>
+          </div>
+          <div class="col-sm-3 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Default simpl-react package for building frontend user interfaces with React and Redux
+            </p>
+          </div>
+          <div class="col-sm-3 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Well documented
+            </p>
+          </div>
+        </div>
+      <h5 class="u-title u-center u-spacerUp">Customizable</h5>
+        <div class="row">
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Game user interfaces can be as basic or as sophisticated as you choose
+            </p>
+          </div>
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Choose between single-player or multi-player
+            </p>
+          </div>
+          <div class="col-sm-4 u-center wow slideInUp" data-wow-offset="140">
+            <p class="tech__points--para">
+              Choose between basic and session authentication
+            </p>
```

Comment:
All games use basic authentication as far as I know.

Reply:
I added a sessions reference into the overview page.